docs/List.md

```diff
@@ -430,6 +430,44 @@ export const PostList = (props) => (
 );
 ```
 
+**Tip**: You can set what route the click on list item redirects to by setting the `redirect` prop.
+
```

Comment:
Please specify all the possible values and their effects

Reply:
it accept's the same as redirect props on edit and create

added the enums to doc